### PR TITLE
Update README regarding to kpm 0.24+ and CNR

### PR DESCRIPTION
**Pull request by Quentin-M**
cc @ant31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,17 +138,19 @@ Technically, cloning Stackanetes is not necessary beside getting the default con
 
 All the configuration is done in one place: the `parameters.yaml` file in the `stackanetes` meta-package. The file is self-documented.
 
-While it is no strictly necessary, it is possible to persist changes to that file for reproducible deployments across environments, without the need of sharing the it out of band. To do this, the stackanetes meta-package has to be pushed, which is currently only possible on a private registry - and because a private registry is used, every packages have to be pushed. Pushing may also be needed when modifications are made to Stackanetes. Here is the simplest way to push the meta-package:
+While it is no strictly necessary, it is possible to persist changes to that file for reproducible deployments across environments, without the need of sharing it out of band. To do this, the stackanetes meta-package has to renamed and pushed to the [CNR Registry](https://cnr.kubespray.io). Pushing is also required when any modifications are made to the Stackanetes packages.
 
     cd stackanetes
-    kpm push -H http://kpm-registry.kpm -f
+    sed -i "s/stackanetes\/stackanetes/<YOUR_CNR_USER_NAME>\/stackanetes/" manifest.jsonnet
```
Comment (ant31) [inline README.md]: It's possible to directly push to a different namespace
eg: 
```
$ kpm push kpm.sh/ant31 
package: ant31/stackanetes (0.1.1 | kpm) pushed

$ kpm deploy kpm.sh/ant31/stackanetes 
-> create stackanetes/stackanetes 

 01 - stackanetes/mariadb:
 --> stacka (namespace): created
 --> mariadb-seedsh (configmap): created
 --> bootstrap-db (configmap): created
 --> mariadb-startsh (configmap): created
[...]
```

Comment (Quentin-M) [inline README.md]: Legit. Updated!

**Comment by mhmxs** [comment]
Hi,
I tested this version on stackanetes, and 3 of the running containers (glance-post, mariadb, nova post) have the same issue. I'm not sure that related to this change. The error message is:
```
bash: /tmp/(post,start).sh: Permission denied
```

**Comment by Quentin-M** [comment]
Users are having issues using Stackanetes with the old README. Merging anyways.

**Pull request merged**